src/crimson/os/seastore/seastore_types.h

```diff
@@ -1362,12 +1398,15 @@ std::ostream &operator<<(std::ostream&, const record_t&);
 struct record_header_t {
   uint32_t deltas;              // number of deltas
   uint32_t extents;             // number of extents
-
+  mod_time_point_t commit_time = 0;
```

Comment:
Why there are two kinds of encoded commit time, `seastar::lowres_system_clock::time_point` in `record_t` and `mod_time_point_t`? Can they be unified?

Reply:
yes, will change it.